File: app/MyJobPitch-New/app/src/main/java/com/myjobpitch/api/data/Image.java
package com.myjobpitch.api.data;

import com.myjobpitch.api.MJPAPIObject;

/**
 * Created by Jamie on 21/05/2015.
 */
public class Image extends MJPAPIObject {
    private String image;
    private String thumbnail;

    public String getImage() {
        return image;
    }

    public String getThumbnail() {
        return thumbnail;
    }
}

File: ios/MyJobPitch/Utils/API/models/Image.swift
//
//  Image.swift
//  MyJobPitch
//
//  Created by dev on 12/20/16.
//  Copyright © 2016 myjobpitch. All rights reserved.
//

import Foundation

class Image: MJPObject {
    
    static var mappingArray = [ "id", "image", "thumbnail" ]

    var image: String!
    var thumbnail: String!
    
}
